### PR TITLE
Added guide-jakarta-websocket to mirror.yml and labMirror.yml

### PR DESCRIPTION
**Pull request by ReeceNana**

### CODE REVIEW

**Review by gkwan-ibm — COMMENTED** [review]
hi @ReeceNana stagingMirror.yml was deprecated. Would you update the labMirror.yml instead?

File: .github/workflows/labMirror.yml [hunk 1]
```diff
@@ -64,6 +64,8 @@ jobs:
           - {"github":"cloud-hosted-guide-rest-intro","gitlab":"IBM-GPXX0JGFEN","gitlab-branch":"lab-483-instruction"}
           - {"github":"cloud-hosted-guide-security-intro","gitlab":"IBM-GPXX06JEN","gitlab-branch":"lab-653-instruction"}
           - {"github":"cloud-hosted-guide-spring-boot","gitlab":"IBM-GPXX0Z2PEN","gitlab-branch":"lab-455-instruction"}
+          - {"github":"cloud-hosted-guide-jakarta-websocket","gitlab":"IBM-GPXX0TEQEN","gitlab-branch":"lab-2183-instruction"}
+
```
Comment (jakub-pomykala) [inline .github/workflows/labMirror.yml:68]: Remove blank line

File: .github/workflows/mirror.yml [hunk 2]
```diff
@@ -65,10 +65,11 @@ jobs:
           - {"github":"cloud-hosted-guide-mongodb-intro","gitlab":"cloud-hosted-guide-mongodb-intro","gitlab_branch":"main"}
           - {"github":"cloud-hosted-guide-graphql-client","gitlab":"cloud-hosted-guide-graphql-client","gitlab_branch":"main"}
           - {"github":"cloud-hosted-guide-grpc-intro","gitlab":"IBM-GPXX0BCEN","gitlab_branch":"main"}
+          - {"github":"cloud-hosted-guide-jakarta-websocket","gitlab":"IBM-GPXX0TEQEN","gitlab_branch":"main"}
 
 
     steps:
-
+f
```
Comment (jakub-pomykala) [inline .github/workflows/mirror.yml]: ???

File: .github/workflows/stagingMirror.yml [hunk 3]
```diff
@@ -69,6 +69,7 @@ jobs:
 
 
 
+
```
Comment (jakub-pomykala) [inline .github/workflows/stagingMirror.yml]: Remove blank line

**Review by jakub-pomykala — CHANGES_REQUESTED** [review]

**Review by jakub-pomykala — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**